components/tag-list/tag-list-item-mixin.js

```diff
@@ -71,24 +108,71 @@ export const TagListItemMixin = superclass => class extends superclass {
 	firstUpdated(changedProperties) {
 		super.firstUpdated(changedProperties);
 
-		const container = this.shadowRoot.querySelector('.tag-list-item-container');
+		const container = this.shadowRoot.querySelector('.tag-list-item-content');
 		this.addEventListener('focus', (e) => {
 			// ignore focus events coming from inside the tag content
-			if (e.composedPath()[0] === this) container.focus();
+			if (e.composedPath()[0] !== this) return;
+			const event = new FocusEvent('focus', { bubbles: true, cancelable: true });
```

Comment:
In order to hopefully improve the screenreader experience (seems to return it to how it was before the tooltip truncation changes + adds in the ability to use the button), this keeps the actual focus on the `d2l-tag-list-item` (or whatever is using the mixin) but then triggers the focus event on the div that's connected to the tooltip, showing the tooltip if truncated.

Let me know any concerns I might be missing around doing things this way.